src/test/kotlin/com/hibob/academy/service/UserServiceTest.kt

```diff
@@ -0,0 +1,94 @@
+package com.hibob.academy.service
+
+import com.hibob.bootcamp.unittests.*
+import org.junit.jupiter.api.Assertions.assertFalse
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.mockito.kotlin.any
+
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.whenever
+
+class UserServiceTest {
+    private val userDao = mock<UserDao>{}
+    private val notificationService = mock<NotificationService>{}
+    private val emailVerificationService = mock<EmailVerificationService>{}
+    private val userService = UserService(userDao, notificationService, emailVerificationService)
+
+    @Test
+    fun `register null user and throws exception`() {
+        val user = User(1 ,"Adi Finkelman", "adifi436@gmail.com", "1234", true)
+        whenever(userDao.findById(user.id)).thenReturn(user)
+        assertThrows<IllegalArgumentException> { userService.registerUser(user) }
```

Comment:
assertThrows returns the exception, so you can also test that the exception message is correct